variables.tf

```
@@ -41,26 +52,31 @@ variable "database_users" {
 # mongodbatlas_advanced_cluster
 
 variable "cluster_name" {
+  description = "Name of the cluster as it appears in Atlas."
   type    = string
   default = "Cluster0"
 }
 
 variable "provider_name" {
+  description = "Cloud service provider on which the servers are provisioned. Possible values are AWS, GCP, AZUER or TENANT."
```

```suggestion
  description = "Cloud service provider on which the servers are provisioned. Possible values are AWS, GCP, AZURE or TENANT."
```

should we explain TENANT?

---

Wanted to follow the same convention as AWS, Oracle, etc. where they put as description a brief sentence. I would suggest explaining it in the global README.md